<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sign_Up</class>
 <widget class="QDialog" name="Sign_Up">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="sign_up_lb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 255, 200); font-size:26pt;</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="email_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_tb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_tb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>340</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="password_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_up_bt">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>540</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 80); font-size:16pt;  color:rgb(200, 255, 200)</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirm_password_tb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="confirm_password_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Confirm Pass</string>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>-20</y>
     <width>591</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/Assets/background.png);
border-radius:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Assets/bg1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickname_tb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14pt; color:rgb(243, 243, 243)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nickname_lb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>141</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15pt; color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
   <property name="indent">
    <number>6</number>
   </property>
  </widget>
  <widget class="QPushButton" name="back_bt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 80); font-size:16pt;  color:rgb(200, 255, 200)</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>sign_up_lb</zorder>
  <zorder>email_lb</zorder>
  <zorder>email_tb</zorder>
  <zorder>password_tb</zorder>
  <zorder>password_lb</zorder>
  <zorder>sign_up_bt</zorder>
  <zorder>confirm_password_tb</zorder>
  <zorder>confirm_password_lb</zorder>
  <zorder>nickname_tb</zorder>
  <zorder>nickname_lb</zorder>
  <zorder>back_bt</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
